services/orchest-webserver/client/src/components/LogViewer.jsx

```diff
@@ -0,0 +1,178 @@
+import React from "react";
```

Comment:
Don't forget type checks!

```suggestion
// @ts-check
import React from "react";
```

Reply:
Good catch.